Release 5.0.0 (#1681)
MetaMask/snaps: packages/snaps-types/CHANGELOG.md

## [Unreleased]

## [0.38.1-flask.1]
### Changed

> - Update transaction insight response and add severity level enum ([#1653](https://github.com/MetaMask/snaps/pull/1653))

Reviewer: ```suggestion
- Update transaction insights response and add severity level enum ([#1653](https://github.com/MetaMask/snaps/pull/1653))
   - Snaps are now able to specify a `severity` for transactions.
   - See [SIP-11](https://metamask.github.io/SIPs/SIPS/sip-11) for more information.
```

## [Unreleased]

## [0.38.1-flask.1]
### Changed
- Update transaction insights response and add severity level enum ([#1653](https://github.com/MetaMask/snaps/pull/1653))

>    - Snaps are now able to specify a `severity` for transactions.

Author: ```suggestion
   - Snaps are now able to specify a `severity` alongside their insight.
```
